src/components/apps/launch/AppLaunchForm.js

```diff
@@ -107,51 +120,194 @@ const StepperBottomNavigation = React.forwardRef((props, ref) => {
+    const {
+        steps,
+        handleStep,
+        handleNext,
+        handleBack,
+        isLastStep,
+        activeStep,
+        formId,
+        errors,
+        touched,
+        groups,
+    } = props;
+    const theme = useTheme();
+    const classes = useStyles();
+    const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
+
+    if (isMobile) {
+        return (
+            <MobileStepper
+                activeStep={activeStep}
+                ref={ref}
+                steps={steps?.length}
+                position="bottom"
+                nextButton={
+                    <Button
+                        size="small"
+                        variant="contained"
+                        onClick={handleNext}
+                        disabled={isLastStep()}
+                        id={buildDebugId(formId, ids.BUTTONS.STEP_NEXT)}
+                        color="primary"
+                    >
+                        {getMessage("next")}
+                        {theme.direction === "rtl" ? (
+                            <KeyboardArrowLeft />
+                        ) : (
+                            <KeyboardArrowRight />
+                        )}
+                    </Button>
+                }
+                backButton={
+                    <Button
+                        size="small"
+                        variant="contained"
+                        onClick={handleBack}
+                        disabled={activeStep === 0}
+                        id={buildDebugId(formId, ids.BUTTONS.STEP_BACK)}
+                    >
+                        {theme.direction === "rtl" ? (
+                            <KeyboardArrowRight />
+                        ) : (
+                            <KeyboardArrowLeft />
+                        )}
+                        {getMessage("back")}
+                    </Button>
+                }
+            />
+        );
+    } else {
+        return (
+            <Stepper
+                alternativeLabel
+                nonLinear
+                activeStep={activeStep}
+                ref={ref}
+                className={classes.stepper}
+            >
+                {steps.map((step, index) => (
+                    <Step key={step.label}>
+                        <StepButton
+                            id={buildDebugId(
+                                formId,
+                                ids.BUTTONS.STEP,
+                                index + 1
+                            )}
+                            onClick={handleStep(index)}
+                        >
+                            <StepLabel
+                                error={
+                                    !!displayStepError(
+                                        index,
+                                        errors,
+                                        touched,
+                                        groups
+                                    )
```

Comment:
This could be passed in as a `stepError` prop to this `LaunchStepper`, which could replace `errors`, `touched`, and `groups` props, since it doesn't look like those are used anywhere else in this component.